src/adapters/webpush/mod.rs

```diff
@@ -172,9 +172,12 @@ impl<C: Controller> Adapter for WebPush<C> {
         &ADAPTER_VERSION
     }
 
-    fn fetch_values(&self, mut set: Vec<Id<Getter>>) -> ResultMap<Id<Getter>, Option<Value>, Error> {
+    fn fetch_values(&self, mut set: Vec<Id<Getter>>, user: User) -> ResultMap<Id<Getter>, Option<Value>, Error> {
         set.drain(..).map(|id| {
-            let user_id = 1; // FIXME: currently logged in user
+            let user_id = match user {
+                User::None => 0,
```

Comment:
Not sure about this one. Should we just bail out? @aosmond ?
Same thing in send_values